### PR TITLE
Always convert contributor githubUsername to url

### PR DESCRIPTION
**Pull request by sandersn**

### CODE REVIEW

File: packages/definitions-parser/src/packages.ts [hunk 1]
```diff
@@ -426,8 +426,10 @@ export class TypingsData extends PackageBase {
       this.data.header.nonNpmDescription ?? unmangleScopedPackage(this.typesDirectoryName) ?? this.typesDirectoryName
     );
   }
-  get contributors(): readonly Owner[] {
-    return this.data.header.owners;
+  get contributors(): readonly Contributor[] {
+    return this.data.header.owners.map((o) =>
+      "githubUsername" in o ? { name: o.name, url: `https://github.com/${o.githubUsername}` } : o
```
Comment (jakebailey) [inline packages/definitions-parser/src/packages.ts:431]: Maybe something like:

```suggestion
      { ...(o.githubUsername ? { url: `https://github.com/${o.githubUsername}` } : undefined), ...o }
```

Old packages had `githubUsername` in them so I'm betting we should keep that?

Comment (sandersn) [reply packages/definitions-parser/src/packages.ts:431]: wellll, I guess so

Comment (jakebailey) [reply packages/definitions-parser/src/packages.ts:431]: This is an awful one liner, a function with a body would probably be better

Comment (sandersn) [reply packages/definitions-parser/src/packages.ts]: Implemented in a more type-system-friendly way though

**Review by jakebailey — APPROVED** [review]

**Pull request merged**